Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/DbController.cs

```diff
@@ -26,42 +24,44 @@ public DbController(MyDbContext context, IConfiguration config)
             _config = config;
         }
 
-          // GET api/values
+        public DbController()
+        {
+        }
+
+        // GET api/values
         [HttpGet]
-        public async Task<ActionResult<IEnumerable<Entry>>> Get()
+        public async Task<ActionResult<IEnumerable<Users>>> Get()
         {
-            return await _db.Entries.ToListAsync();
+            return await _db.UsersEntries.ToListAsync();
         }
 
         // GET api/values/5
         [HttpGet("{id}")]
         public async Task<IActionResult> GetEntry([FromRoute] int id)
         {
-            var entry = await _db.Entries.SingleOrDefaultAsync(m => m.Id == id);
+            var entry = await _db.UsersEntries.SingleOrDefaultAsync(m => m.UsersId == id);
             if (entry == null)
             {
                 return NotFound();
-
             }
+            _db.UsersEntries.Remove(entry);
```

Comment:
Think I asked this before. But what is the reason for removing the user entry from the collection when doing a get request?

Reply:
This method currently looks very much like the deleteEntry Task

Reply:
DBcontroller was just for testing purposes, will clear up and redo all controllers in my next sprint